### PR TITLE
Extract translations

### PR DESCRIPTION
**Pull request by reinhardt**
syslabcom/scrum#1137

### CODE REVIEW

**Comment by reinhardt** [comment]
Tests are failing but that's unrelated to this pull request. It's fine to merge. We're working on the tests.

File: src/osha/oira/locales/bg/LC_MESSAGES/euphorie.po [hunk 1]
```diff
@@ -23,6 +23,10 @@ msgstr ""
 msgid "${certificate/title}"
 msgstr ""
 
+#: osha.oira/src/osha/oira/client/browser/templates/dashboard-banner.pt:15
+msgid "${message/img_alt}"
+msgstr ""
+
```
Comment (ale-rt) [inline src/osha/oira/locales/bg/LC_MESSAGES/euphorie.po:29]:
```suggestion
```
This shows an error in the template.

I will fix it, but I will not merge this bogus `msgid` for the moment.

Comment (ale-rt) [reply src/osha/oira/locales/bg/LC_MESSAGES/euphorie.po:29]: Removed here https://github.com/euphorie/osha.oira/pull/212

**Comment by reinhardt** [comment]
rebased and reextracted

**Review by ale-rt — APPROVED** [review]

**Pull request merged**